clients/pkg/promtail/client/manager.go

```diff
@@ -64,12 +82,8 @@ func (m *Manager) start() {
 	m.wg.Add(1)
 	go func() {
 		defer m.wg.Done()
-		// keep reading received entries
-		for e := range m.entries {
-			// then fanout to every remote write client
-			for _, c := range m.clients {
-				c.Chan() <- e
-			}
+		// discard read entries
+		for range m.entries {
```

Comment:
what's the reason for throwing them away?

Reply:
Until we refactor all types of clients inside the manager, `client.Manager` is responsible just for the WAL enabled use case. Since in this PR we are still in the latter case (without refactoring), and we are implementing the wal reader side which would read all entries from the WAL and deliver them to each remote write client, we don't need the manager to also write them to the client.

That bit of code was in place for making the wal with just the writer side implemented work.

Reply:
we should maybe just remove the entries channel then? and then start the watcher here + stop it in the `manager.Stop` function?

Reply:
oh right, we need to keep this since the writer fanout writer has to take in things that satisfy the entry handler interface

Reply:
Exactly! On the refactor PR (which will be the following), I can take another pass at this and see if there's some way to simplify it